mirdata/beatport_key.py

```diff
@@ -0,0 +1,348 @@
+
+Ángel Faraldo (2017). Tonality Estimation in Electronic Dance Music: A Computational and Musically Informed
+Examination. PhD Thesis. Universitat Pompeu Fabra, Barcelona.
+
+This dataset is mainly intended to assess the performance of computational key estimation algorithms in electronic
+dance music subgenres.
+
+Data License: Creative Commons Attribution Share Alike 4.0 International
+"""
+import fnmatch
+import json
+import librosa
+import os
+
+from mirdata import download_utils
+from mirdata import jams_utils
+from mirdata import track
+from mirdata import utils
+
+DATASET_DIR = 'beatport_key'
+REMOTES = {
+    'keys': download_utils.RemoteFileMetadata(
+        filename='keys.zip',
+        url='https://zenodo.org/record/1101082/files/keys.zip?download=1',
+        checksum='939abc05f36121badfac4087241ac172',
+        destination_dir='.',
+    ),
+    'metadata': download_utils.RemoteFileMetadata(
+        filename='original_metadata.zip',
+        url='https://zenodo.org/record/1101082/files/original_metadata.zip?download=1',
+        checksum='bb3e3ac1fe5dee7600ef2814accdf8f8',
+        destination_dir='.',
+    ),
+    'audio': download_utils.RemoteFileMetadata(
+        filename='audio.zip',
+        url='https://zenodo.org/record/1101082/files/audio.zip?download=1',
+        checksum='f490ee6c23578482d6fcfa11b82636a1',
+        destination_dir='.',
+    ),
+}
+
+DATA = utils.LargeData('beatport_key_index.json')
+
+
+class Track(track.Track):
+    """beatport_key track class
+    Args:
+        track_id (str): track id of the track
+        data_home (str): Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets`
+    Attributes:
+        audio_path (str): track audio path
+        keys_path (str): key annotation path
+        metadata_path (str): sections annotation path
+        title (str): title of the track
+        track_id (str): track id
+    """
+
+    def __init__(self, track_id, data_home=None):
+        if track_id not in DATA.index:
+            raise ValueError(
+                '{} is not a valid track ID in beatport_key'.format(track_id)
+            )
+
+        self.track_id = track_id
+
+        if data_home is None:
+            data_home = utils.get_default_dataset_path(DATASET_DIR)
+
+        self._data_home = data_home
+        self._track_paths = DATA.index[track_id]
+        self.audio_path = os.path.join(self._data_home, self._track_paths['audio'][0])
+        self.keys_path = os.path.join(self._data_home, self._track_paths['key'][0])
+        self.metadata_path = (
+            os.path.join(self._data_home, self._track_paths['meta'][0])
+            if self._track_paths['meta'][0] is not None
+            else None
+        )
+        self.title = self.audio_path.replace(".mp3", '').split('/')[-1]
+
+    @utils.cached_property
+    def key(self):
+        """String: key annotation"""
```

Comment:
I've noticed we're mixing docs styles. I don't know if this is the only loader that does it, but I'll add it as part of the audit issue to make sure we are consistent later on

Reply:
I was imitating beatles loader. What style must we use?